tests/main/views/test_feedback.py

```diff
@@ -30,9 +29,3 @@ def test_google_gone_logs_problem(self, current_app, external_requests_post):
     def test_feedback_submission(self, external_requests_post):
         external_requests_post.return_value.status_code = 200
         self._check_success(self._post())
-
-    def test_feedback_form_has_csrf(self):
```

Comment:
I wonder if we should return `404` at the `/feedback` view for now?

Reply:
What would be the best way to do that, taking out the view altogether?

Reply:
Strictly speaking it should be a `410 Gone`, as we're retiring the URL. We have a template for that already - let's keep the `send_feedback` view but replace all the code with `abort(410)`.

Reply:
👍 great that sounds like a plan

Reply:
I've put together a commit to do this